spec/rock_rms/resources/recurring_donation_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'spec_helper'
+
+RSpec.describe RockRMS::Client::RecurringDonation, type: :model do
+  include_context 'resource specs'
+
+  describe '#list_recurring_donations(options = {})' do
+    it 'returns a array of hashes' do
+      resource = client.list_recurring_donations
+      expect(resource).to be_a(Array)
+      expect(resource.first).to be_a(Hash)
+    end
+
+    it 'queries recurring donations' do
+      expect(client).to receive(:get)
+        .with('FinancialScheduledTransactions', {})
+        .and_call_original
+      client.list_recurring_donations
+    end
+
+    it 'formats with RecurringDonation' do
+      response = double
+      expect(RockRMS::Responses::RecurringDonation).to receive(:format).with(response)
```

Comment:
Need specs for `RockRMS::Responses::RecurringDonation` for this expectation to be meaningful.

Reply:
Just added. A spec for the formatter.